index.js

```diff
@@ -30,11 +25,14 @@ AnimatedGifDetector.prototype.isAnimated = function(buffer) {
   var result = false
     , count = 0
   ;
-  for (var i = 0; i < buffer.length; i++) {
+  var packed = buffer.slice(10, 11)[0];
```

Comment:
re: your comment, this should be that whole packed byte, i believe. which means that this does indeed have an issue.

I can follow on with a fix, but I may have to manufacture a gif to expose the bug, heh.